### PR TITLE
Selectors for Arithmetic Inline + Arithmetic Inline AT almost done

### PR DESCRIPTION
**Pull request by dashasandler**
https://trello.com/c/5C2WsMmL/848-at25javascript-entity-arithmetic-inline
Please check it one more time I made a mistake and did my Automation test in same branch so right now technically it is completely different pull request.

### CODE REVIEW

**Review by Burdishka — APPROVED** [review]

**Review by MariaMoroz — APPROVED** [review]

**Review by DalidovichM — APPROVED** [review]

File: test/specs/arithmeticInline.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        $(selectors.navigationBar.sideBar).click();
+        $(selectors.arithmeticInline.sidebarArithmeticInline).scrollIntoView();
+        $(selectors.arithmeticInline.sidebarArithmeticInline).click();
+
+        expect($(selectors.arithmeticInline.mainPanel).getText())
+            .to.equal(expected.arithmeticInline.mainPanelText);
+    });
+    it('02 Should verify h3 text equal to the Arithmetic Inline', () => {
+        expect($(selectors.arithmeticInline.h3Text).getText())
+            .to.equal(expected.arithmeticInline.mainPanelText);
```
Comment (Burdishka) [inline test/specs/arithmeticInline.js:25]: I am pretty sure this verification would be great to use in '01 Should go to the Arithmetic Inline' after first verification after the 21st line

File: test/specs/arithmeticInline.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    });
+    it('03 Should click Create new record Btn for simple Save', () => {
+        $(selectors.arithmeticInline.createNewRecord).click();
+        expect($(selectors.arithmeticInline.h4).getText())
+            .to.equal(expected.arithmeticInline.h4);
+    });
+    it('04 Should be able to fill fields for simple Save', () => {
+
```
Comment (Burdishka) [inline test/specs/arithmeticInline.js:33]: unnecessary empty line

File: test/specs/arithmeticInline.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+                    .to.equal(expected.arithmeticInline.expectedValues[el]);
+            }
+        });
+    it('9A Should  delete record from list', () => {
+        $(selectors.arithmeticInline.listBtn).click();
+        $(selectors.arithmeticInline.actionsDropdownBtn).click();
+        $(selectors.arithmeticInline.delete).click();
+        expect(getRowCount()).to.equal(expected.arithmeticInline.zeroRec);
+    });
+
```
Comment (Burdishka) [inline test/specs/arithmeticInline.js:86]: empty line

File: test/specs/arithmeticInline.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    it('10 Should click Create new record Btn for Save as Draft', () => {
+        $(selectors.arithmeticInline.createNewRecord).click();
+        expect($(selectors.arithmeticInline.h4).getText())
+            .to.equal(expected.arithmeticInline.h4);
+    });
+    it('11 Should be able to fill fields for Save as draft', () => {
+
```
Comment (Burdishka) [inline test/specs/arithmeticInline.js:93]: empty line

File: test/specs/arithmeticInline.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    it('16 Should check that table on the Order mode was filled with ' +
+        'correct values for save as draft',() => {
+        for(let el = 0; el < $(selectors.arithmeticInline.table).length; el++) {
+            expect($(selectors.arithmeticInline.table)[el].getText())
+                .to.equal(expected.arithmeticInline.expectedValues[el]);
+        }
+    });
+    it('17 Should delete record from List mode', () => {
+        $(selectors.arithmeticInline.listBtn).click();
+
```
Comment (Burdishka) [inline test/specs/arithmeticInline.js:143]: empty line

File: test/specs/arithmeticInline.js [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+});
+});
```
Comment (Burdishka) [inline test/specs/arithmeticInline.js:145]: 146 line is needed in this case
@SergeiKuznetsov would be great if you explain WHY!

**Review by Burdishka — COMMENTED** [review]
I am not sure but looks like this branch was not deleted and work was continued in the old branch after PR was merged.
Every next PR should start in the new branch, and after merging should be deleted.

It would be great if any date or selectors that can be used by any engineers would be located in common

**Comment by Burdishka** [comment]
if you see you don't need any reviews anymore)))) so after this merge, you should delete this branch here and locally also you don't need it. Just create new one when you start new task

**Review by gmirzoyan — APPROVED** [review]
Sorry, I accidentally pushed on "close pull request" button, but after realizing it I re-opened PR again.
All tests passed and final code looks good for me.

File: test/specs/arithmeticInline.js [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+import testData from "./../../data/testData.json";
+import expected from "./../../data/expected.json";
+
+describe('ARITHMETIC INLINE', () => {
+    before(() => {
+        browser.maximizeWindow();
+        browser.url('/');
+        userLogin(testData.userLogin.loginName, testData.userLogin.password);
+        reset();
+        browser.pause(3000)
```
[the remark this answers is not in the document]
Comment (dashasandler) [reply test/specs/arithmeticInline.js]: Thanks, forgot to delete!

File: test/specs/arithmeticInline.js (hunk 1 above)
Comment (dashasandler) [reply test/specs/arithmeticInline.js:23]: This is two different verification, I have 2 different elements with same names on one page and decided to split it just for visualization

**Pull request merged**